regression-test/suites/external_table_p0/jdbc/test_oracle_jdbc_catalog.groovy

```diff
@@ -215,15 +215,15 @@ suite("test_oracle_jdbc_catalog", "p0,external,oracle,external_docker,external_d
         qt_specified_database   """ show databases; """
         sql """drop catalog if exists ${catalog_name} """
 
-        // test lower_case_table_names argument
+        // test lower_case_meta_names argument
         sql """create catalog if not exists ${catalog_name} properties(
                     "type"="jdbc",
                     "user"="doris_test",
                     "password"="123456",
                     "jdbc_url" = "jdbc:oracle:thin:@${externalEnvIp}:${oracle_port}:${SID}",
                     "driver_url" = "${driver_url}",
                     "driver_class" = "oracle.jdbc.driver.OracleDriver",
-                    "lower_case_table_names" = "true"
```

Comment:
better to leave a case with `lower_case_table_names`, to test compatibility

Reply:
Added error reporting test for old property